### ISSUE TITLE
BOLT2: MUST/MAY rejection of a `open_channel` attempt

### ISSUE THREAD

**Issue by C-Otto** [lightningnetwork/lightning-rfc#420]
Currently a receiver MUST fail a channel if "`to_self_delay` is unreasonably large". As this is not properly defined, I would prefer to rephrase this as a MAY.

**Comment by C-Otto** [comment]
Similar: "it considers `feerate_per_kw` too small for timely processing or unreasonably large."

**Comment by rustyrussell** [comment]
I understand your comment, however, it's still a MUST, though poorly defined.

You *MUST* have some limit you on what you will accept.  We reluctantly don't define that limit, however, because we don't have enough experience to do so.
